stories/Organism/Images/Images/images.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../../assets/scss/variables';
+@import '../../../assets/scss/mixins';
+@import '../../../assets/scss/base';
+
+/* images start */
+.image {
+  &__cart {
+    img {
+      @extend %img-cover;
+
+      display: block;
+    }
+  }
+
+  &__description {
+    .image__caption {
+      margin: $spacing-16 $spacing-24 0;
+
+      @include devicebreak(medium) {
+        margin: $spacing-32 0 0;
+      }
+    }
+  }
+
+  &-section {
+    justify-content: center;
+
+    &--portrait {
+      align-items: flex-end;
+
+      @include devicebreak(medium) {
+        .image__caption {
+          display: block;
+          margin: 0 0 0 $spacing-24;
+          padding-bottom: 0;
+
+          &_para {
+            font-size: $font-size-14;
+            line-height: 1.203rem;
```

Comment:
@Shubham-Sharma-77 Can we use the unitless as it will calculate the line height according to the font size

Reply:
No because the line height of the body is different from this paragraph.

Reply:
@Shubham-Sharma-77 still we cannot use rem in line-height and you have to adjust line-height as unitless and we can do that